### PR TITLE
Feature/pd 9 responsive development

### PR DESCRIPTION
**Pull request by silent-improvement**

### CODE REVIEW

File: scss/partials/_content.scss [hunk 1]
```diff
@@ -30,26 +45,23 @@
...
+    @include breakpoint-sm-down {
+      font-size: 0.8rem;
+    }
   }
 
   &__button-wrapper {
     height: 6rem;
 
-    @include large-container;
+    @include x-large-container;
```
Comment (phillygogo) [inline scss/partials/_content.scss]: Remove unneeded code

File: scss/partials/_content.scss [hunk 2]
```diff
@@ -84,7 +96,7 @@
   }
 
   &__portrait {
```
Comment (phillygogo) [inline scss/partials/_content.scss]: remove unused Portrait

File: scss/partials/_content.scss [hunk 3]
```diff
@@ -5,9 +5,19 @@
   align-items: center;
   flex-direction: column;
 
-  @include large-container;
+  @include x-large-container;
   padding-top: 5rem;
 
+  @include breakpoint-lg-down {
```
Comment (phillygogo) [inline scss/partials/_content.scss]: Duplicate code here too, unneeded breakpoints. Set a bunch of default styles then only change the things you need when going down through the breakpoints,

File: pages/about.html [hunk 4]
```diff
@@ -77,11 +79,12 @@ <h3 class="header__title">
         <div class="content__title">About Silent Improvement</div>
         <div class="content__content">
```
Comment (phillygogo) [inline pages/about.html]: This isn't correct BEM. Maybe it should be `content__content-wrapper`?

File: index.html [hunk 5]
```diff
@@ -118,7 +120,7 @@ <h3 class="header__title">
                     <img src="https://www.citypng.com/public/uploads/preview/-51609516149nowldl3ocn.png"
                         class="card-button__image" alt=""></img>
                     <div class="card-button__content-wrapper">
-                        <span class="card-button__content">
+                        <span class="card-button__content cantarell-regular">
```
Comment (phillygogo) [inline index.html]: use a mixin instead of adding classes to every row. Remove all these classes for fonts and replace with the new way I showed you

**Review by phillygogo — CHANGES_REQUESTED** [review]

File: index.html [hunk 6]
```diff
@@ -21,8 +25,8 @@
                 <img src="/assets/logo.png" class="header__logo" alt="Silent Improvement Logo">
                 </img>
             </a>
-            <h3 class="header__title">
-                <a href="/index.html">
+            <h3>
```
Comment (phillygogo) [inline index.html]: test this with screen reader, see how it reads it out. then try changing it tso the H3 is in the anchor

File: scss/partials/_header.scss [hunk 7]
```diff
@@ -152,6 +187,12 @@
       color: #d89026;
     }
 
+    @include breakpoint-ml-down {
```
Comment (phillygogo) [inline scss/partials/_header.scss]: Review hovers on tablet and mobile

File: scss/partials/_header.scss [hunk 8]
```diff
@@ -152,6 +187,12 @@
...
+      &:hover {
+        color: #342e2d;
+      }
+    }
+
     @include breakpoint-sm-down {
```
Comment (phillygogo) [inline scss/partials/_header.scss]: remove hovers on Tablet and Mobile

File: scss/_utilities.scss [hunk 9]
```diff
@@ -24,20 +24,145 @@
...
-  padding: 0 1rem;
-  margin: auto;
+}
+
+@mixin container-fluid {
+  padding-inline: 1rem;
+  margin: 0 auto;
+}
+
+@mixin x-large-heading() {
```
Comment (phillygogo) [inline scss/_utilities.scss]: move all typography related stuff out of _utilities  and put them in the Typography file.

Then move all Utilities out, and put them in the Utilities file.

File: scss/_utilities.scss [hunk 10]
```diff
@@ -24,20 +24,145 @@
...
+    @include text-small;
+    font-weight: 400;
+  }
+}
+
+@mixin text-small {
+  @include text-small;
+  font-weight: 400;
+
+  @include breakpoint-lg-down() {
```
Comment (phillygogo) [inline scss/_utilities.scss]: unneeded

File: scss/_utilities.scss [hunk 11]
```diff
@@ -24,20 +24,145 @@
...
+    @include text-medium;
+    font-weight: 400;
+  }
+
+  @include breakpoint-ml-down() {
+    @include text-small;
+    font-weight: 400;
+  }
+
+  @include breakpoint-sm-down() {
```
Comment (phillygogo) [inline scss/_utilities.scss]: unneeded

**Review by phillygogo — CHANGES_REQUESTED** [review]

File: index.html (hunk 6 above)
Comment (silent-improvement) [reply index.html]: Tested with screen reader and made change after reading about best practice

File: scss/partials/_header.scss [hunk 12]
```diff
@@ -3,23 +3,26 @@
...
+    @include breakpoint-ml-down {
+      display: flex;
+      align-items: center;
+      justify-content: flex-end;
+      border: none;
+      background-color: #1c1919;
+    }
+
     @include breakpoint-sm-down {
       display: flex;
```
Comment (phillygogo) [inline scss/partials/_header.scss:13]: remove duplicate

File: scss/partials/_footer.scss [hunk 13]
```diff
@@ -43,16 +40,44 @@
...
   .fa-instagram,
   .fa-youtube {
     font-size: 2.625rem;
     padding-bottom: 1rem;
+
+    @include breakpoint-ml-down {
+      padding-bottom: 3rem;
+    }
+
+    @include breakpoint-sm-down {
```
Comment (phillygogo) [inline scss/partials/_footer.scss]: unneeded duplicates through breakpoints

File: scss/partials/_header.scss [hunk 14]
```diff
@@ -164,8 +199,14 @@
...
     fill: #fafafa;
 
+    @include breakpoint-ml-down {
+      width: 1.5rem;
+      margin-right: 3rem;
+    }
+
     @include breakpoint-sm-down {
       width: 1.5rem;
+      margin-right: 3rem;
```
Comment (phillygogo) [inline scss/partials/_header.scss:163]: not needed duplicate

File: scss/partials/_header.scss [hunk 15]
```diff
@@ -179,38 +220,54 @@
...
 
   &__menu-accordion {
+
+    @include breakpoint-ml-down {
+      width: 100%;
+    }
+
     @include breakpoint-sm-down {
-      width: 87%;
+      width: 100%;
```
Comment (phillygogo) [inline scss/partials/_header.scss:193]: dupliacate

File: scss/partials/_header.scss [hunk 16]
```diff
@@ -179,38 +220,54 @@
...
+      border: unset;
+      background-color: #1c1919;
+      padding: 0.5rem 3.5rem 0.5rem 9rem;
+      margin-top: -3rem;
+      border-radius: unset;
+    }
 
     @include breakpoint-sm-down {
       width: 100%;
```
Comment (phillygogo) [inline scss/partials/_header.scss]: duplicate

**Review by phillygogo — CHANGES_REQUESTED** [review]

**Pull request merged**